commands.txt

```diff
@@ -1,6 +1,8 @@
-# Build every buildable Bazel target and test every testable target
-# TODO: Until we move to Angular Bazel, execute `npm install` beforehand
-bazel query //... | grep -v ^\/\/third_party | grep -v *node_modules | xargs bazel test
+# Build every buildable Bazel target
```

Comment:
I think we still need to ignore `npm install` because to run Angular server, it's still needed, and so will still create the same `node_modules` directory. What we don't need anymore is the `npm install` before we run `bazel build`. So I think the only change here is removing the TODO.